contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -2165,6 +2166,7 @@ babelfish_add_domain_mapping_entry_internal(PG_FUNCTION_ARGS)
 		FlushErrorState();
 		MemoryContextSwitchTo(ectx);
 
+		RESUME_INTERRUPTS();
 		ereport(ERROR,
```

Comment:
It is kind of re-throw only, we are re-throwing new error here because we would like to make it clear that operation to add domain mapping entry is failed. Without this, we may get error such as "duplicate key violation" from engine which may not make much sense to end user.

Reply:
is it safe to throw a new error in CATCH without HOLD, wouldn't it run into parallel connection issue?